test/vm-download

```diff
@@ -102,7 +96,9 @@ def prune():
             targets.append(target)
     for filename in os.listdir(DATA):
         path = os.path.join(DATA, filename)
-        if os.path.isfile(path) and path.endswith(".qcow2") and path not in targets:
+        if not force and os.stat(path).st_mtime > now - testinfra.IMAGE_EXPIRE * 86400:
```

Comment:
This only deletes images that have been pulled, wouldn't they just get pulled again when needed? The real interesting one is in vm-upload, which deletes the "original" copy. The idea is that between them, if someone really wants to keep an image we'll be able to find a copy.